src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public class Node<K , V>{
+        private int hash;
+        private K key;
+        private V value;
+        Node<K, V> next;
+
+
+        public Node(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
+    private  Node<K, V> head;
+
+    public static final int DEFAULT_CAPACITY = 16;
+
+    public static final float LOAD_FACTOR = 0.75f;
+
+    private int currentCapacity = DEFAULT_CAPACITY;
+
+    private int threshold = (int) (currentCapacity * LOAD_FACTOR);
+
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+
+    private int count;
 
     @Override
     public void put(K key, V value) {
+        Node<K, V> newNode = new Node<>(key,value);
```

Comment:
The `newNode` is created, but the hash value is not set. Consider setting the hash value using the key's hash code.